index.spec.js

```diff
@@ -0,0 +1,67 @@
+const request = require('supertest')
+
+const server = require('./index.js')
+
+
+describe('server.js', () => {
+  describe('/ route', () => {
+    it('should return status code 200', async () => {
+      let response = await request(server).get('/');
+      expect(response.status).toBe(200)
+    })
+
+    it('should return a json object', async () => {
+      let response = await request(server).get('/')
+      expect(response.type).toBe('application/json')
+    })
+  })
+
+  describe('/hello route', () =>{
+    it('should return with a body like this {message: hello}', async ()=> {
+      let response = await request(server).get('/hello')
+      expect(response.body).toEqual({message: 'hello'})
+
+    })
+
+    it('should return status code 200', async () => {
+      let response = await request(server).get('/hello');
+      expect(response.status).toBe(200)
```

Comment:
That test fails because You are not passing any object to the post request, hence server will throw `500`.